common/internal_model/src/main/scala/weco/catalogue/internal_model/identifiers/CanonicalID.scala

```diff
@@ -0,0 +1,56 @@
+package weco.catalogue.internal_model.identifiers
+
+import io.circe.{Decoder, Encoder, HCursor, Json}
+import org.scanamo.DynamoFormat
+
+class CanonicalID(val underlying: String) {
```

Comment:
This is such a bikeshed but I'm going to do it - I think we use `CamelCaseId` everywhere else, is there a reason for this being `CamelCaseID`?

Reply:
🚲 🏠 

Because I forgot we use camel case everywhere.

Also, we use `IngestID` in the storage service, which is the code I was using as a reference point for this work. I'll change it. https://github.com/wellcomecollection/storage-service/blob/122c03d0e90007fcc0781376dcf79e77bb5c388a/common/src/main/scala/uk/ac/wellcome/platform/archive/common/ingests/models/IngestID.scala